test/data.js

```diff
@@ -37,6 +37,38 @@ test( "jQuery.data() camelCased names", function( assert ) {
 		} );
 	} );
 
+	div = document.createElement( "div" );
```

Comment:
Could you move all this code to a separate `test`? An example name: "jQuery.data() camelCased names (mass setter)".